tensorflow/java/src/main/native/saved_model_bundle_jni.cc

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <limits>
+#include <memory>
+
+#include "tensorflow/c/c_api.h"
+#include "tensorflow/java/src/main/native/exception_jni.h"
+#include "tensorflow/java/src/main/native/saved_model_bundle_jni.h"
+
+JNIEXPORT jobject JNICALL Java_org_tensorflow_SavedModelBundle_load(JNIEnv * env,
+                                                                    jclass clazz,
+                                                                    jstring export_dir,
+                                                                    jobjectArray tags,
+                                                                    jbyteArray run_options) {
+
```

Comment:
I just realized that this will fail on the Android tests since we've currently disabled `TF_LoadSessionFromSavedModel` in Android (since it pulls it a bit too many dependencies right now).
I'll make a change to `c_api.h` so that the symbol will exist, even if the call will fail on Android.

So no changes needed in this PR for that, but just a heads up.